pkg/roachprod/roachprod.go

```diff
@@ -826,6 +826,32 @@ func updatePrometheusTargets(ctx context.Context, l *logger.Logger, c *install.S
 	}
 }
 
+// regionRegEx is the regex to extract the region label from zone available as vm property
+var regionRegEx = regexp.MustCompile("(^.+[0-9])(-[a-f]$)")
```

Comment:
I had to check, but it does look like some of the GCE regions [1] are _now_ in double digits, e.g., `europe-west12-a` :)

[1] https://cloud.google.com/compute/docs/regions-zones

Reply:
actually, .* is absorbing the digit as well in this case. I will make it to [0-9]+ to be safer.

Reply:
Thanks for pointing this out!